src/alfresco-core-rest-api/src/api/NodesApi.js

```diff
@@ -397,7 +398,8 @@
       return this.apiClient.callApi(
         '/nodes/{nodeId}/children', 'GET',
         pathParams, queryParams, headerParams, formParams, postBody,
-        authNames, contentTypes, accepts, returnType
+        authNames, contentTypes, accepts, returnType, undefined,
```

Comment:
What do you mean by "main call"?  We need some way for the requester to pass the value in for each request.